docs/config/keys.md

```diff
@@ -100,6 +100,24 @@ return {
 }
 ```
 
+### Microsoft Windows and Ctrl-Alt <-> AltGr
+
+*since: nightly*
+
+If you are using a layout with an *AltGr* key, you may experience issues
```

Comment:
Please also add a `docs/config/lua/config/treat_left_ctrlalt_as_altgr.md` with this information inside it.

It's unfortunate to duplicate the content in two places, and this particularly page is already quite long, so I'd suggest moving this block into that separate page, and then here in this doc you can have a shorter paragraph that says:

> If you are using VNC and a keyboard layout with dead keys, then you may wish to enable \[treat_left_ctrlalt_as_altgr\](lua/config/treat_left_atrlalt_as_altgr.md).

Please also add a line to `docs/changelog.md` summarizing this PR and linking to that new doc page!

Reply:
Done!